libraries/Microsoft.Bot.Builder.AI.QnA/Dialogs/QnAMakerDialog.cs

```diff
@@ -289,6 +289,8 @@ public QnAMakerDialog([CallerFilePath] string sourceFilePath = "", [CallerLineNu
         /// <seealso cref="RankerTypes"/>
         [JsonProperty("rankerType")]
         public StringExpression RankerType { get; set; } = new StringExpression(RankerTypes.DefaultRankerType);
+      
+        public float MaximumScoreForLowScoreVariation { get => maximumScoreForLowScoreVariation; set => maximumScoreForLowScoreVariation = value; }
```

Comment:
>MaximumScoreForLowScoreVariation [](start = 21, length = 32)

missing comments
missing JsonProperty attribute
should probably be a ExpressionProperty<float>

Reply:
Implemented.